tests/codebuild/run_all_sample_canary_tests.sh

```diff
@@ -6,10 +6,10 @@ source delete_tests.sh
 source inject_tests.sh
 source smlogs_tests.sh
 
-run_canary_tests
-run_feature_canary_tests
-verify_canary_tests
-verify_feature_canary_tests
-run_smlogs_canary_tests
-delete_all_resources # Delete all existing resources to re-use metadata names
-run_delete_canary_tests
+run_canary_tests "default"
+run_feature_canary_tests "default"
+verify_canary_tests "default"
+verify_feature_canary_tests "default"
+run_smlogs_canary_tests "default"
+delete_all_resources "default" # Delete all existing resources to re-use metadata names
+run_delete_canary_tests "default" 
```

Comment:
This seems like it is redundant if we could otherwise just set a global variable (overrided by an env variable) and each of the functions would assume this as the default namespace, instead.

Reply:
Umm I would not be worry so much on this. That might break readability but good suggestion though,